From ab8b21261427283791f847675bb8512a46782838 Mon Sep 17 00:00:00 2001
From: Mathew McBride <matt@traverse.com.au>
Date: Wed, 2 Jul 2025 13:03:04 +1000
Subject: [PATCH 4/4] platform: rasbperry-pi: remove functions now implemented
 in main HAL

platform_pi.c:644: multiple definition of `wifi_rrm_send_beacon_req'
platform_pi.c:652: multiple definition of `wifi_wnm_send_bss_tm_req'
platform_pi.c:658: multiple definition of `handle_wnm_action_frame'
platform_pi.c:664: multiple definition of `handle_rrm_action_frame'

Change-Id: I1c3c2ddcf76522a1387afe487cddbfeea0641dc3
Signed-off-by: Mathew McBride <matt@traverse.com.au>
---
 platform/raspberry-pi/platform_pi.c | 30 -----------------------------
 1 file changed, 30 deletions(-)

diff --git a/platform/raspberry-pi/platform_pi.c b/platform/raspberry-pi/platform_pi.c
index 93b8bd7..fca35a1 100644
--- a/platform/raspberry-pi/platform_pi.c
+++ b/platform/raspberry-pi/platform_pi.c
@@ -635,36 +635,6 @@ INT wifi_steering_eventRegister(wifi_steering_eventCB_t event_cb)
     return RETURN_OK;
 }
 
-int wifi_rrm_send_beacon_req(struct wifi_interface_info_t *interface, const u8 *addr,
-    u16 num_of_repetitions, u8 measurement_request_mode, u8 oper_class, u8 channel,
-    u16 random_interval, u16 measurement_duration, u8 mode, const u8 *bssid,
-    struct wpa_ssid_value *ssid, u8 *rep_cond, u8 *rep_cond_threshold, u8 *rep_detail,
-    const u8 *ap_ch_rep, unsigned int ap_ch_rep_len, const u8 *req_elem, unsigned int req_elem_len,
-    u8 *ch_width, u8 *ch_center_freq0, u8 *ch_center_freq1, u8 last_indication)
-{
-    return 0;
-}
-
-/* called by BTM API */
-int wifi_wnm_send_bss_tm_req(struct wifi_interface_info_t *interface, struct sta_info *sta,
-    u8 dialog_token, u8 req_mode, int disassoc_timer, u8 valid_int, const u8 *bss_term_dur,
-    const char *url, const u8 *nei_rep, size_t nei_rep_len, const u8 *mbo_attrs, size_t mbo_len)
-{
-    return 0;
-}
-
-int handle_wnm_action_frame(struct wifi_interface_info_t *interface, const mac_address_t sta,
-    struct ieee80211_mgmt *mgmt, size_t len)
-{
-    return 0;
-}
-
-int handle_rrm_action_frame(struct wifi_interface_info_t *interface, const mac_address_t sta,
-    const struct ieee80211_mgmt *mgmt, size_t len, int ssi_signal)
-{
-    return 0;
-}
-
 INT wifi_setApManagementFramePowerControl(INT apIndex, INT dBm)
 {
     return 0;
-- 
2.39.5

